youtube-ios/Example/InlineYoutubeView/YTViewController.m

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
we would have to add license header to the demo files also

Reply:
Have added //  Copyright © 2018 flipkart.com. All rights reserved. Shouldn't this suffice?

Reply:
let's add the apache licence text also, similar to the one added in library files

Reply:
Added the LICENSE text. Regarding the the 2nd license for flipkart, should I add another LICENSE file with a different name?